src/libraries/ExcelConverter/Utils.cs

```diff
@@ -252,6 +254,19 @@ public static String ReformatRange(String input)
             return colonRangeRegex.Replace(input, "$1_RANGE_$2");
         }
 
+        // Takes in a range eg. "A4:C9" and returns its corresponding rang eobject
+        public static Range DecomposeRange(String strInput)
+        {
+            Match match = decomposeColonRangeRegex.Match(strInput);
+
+            if (!match.Success) return null;
+
+            Range output = new Range(char.Parse(match.Groups[1].Value), int.Parse(match.Groups[2].Value), 
+                char.Parse(match.Groups[3].Value), int.Parse(match.Groups[4].Value));
```

Comment:
you can use just the Match constructor here instead

Reply:
will resolve later during refactoring phase